app/templates/buyers/index.html

```diff
@@ -16,19 +16,19 @@
 
 {% block main_content %}
 <div class="grid-row">
-  <div class="column-full">
+  <div class="column-two-thirds">
     {% with
         context = current_user.email_address,
         heading = current_user.name
     %}
     {% include 'toolkit/page-heading.html' %}
     {% endwith %}
   </div>
-  <div class="column-full dm-account-overview-options">
+  <div class="column-two-thirds dm-account-overview-options">
     
     <div class="framework">
```

Comment:
We should still be able to do this without a specific framework div as there isn't really anything special about the content here.

Reply:
Yeah we can get the styling we want without using extra divs by just having everything in a single dmspeak div.

Reply:
If we've added SCSS somewhere to support the frameworks div, we should remove that.

Reply:
ok sure I'll get those changes in